app/tests/spec/views/coppa/coppa-age-input.js

```diff
@@ -67,6 +67,24 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('onKeyDown', function () {
+      function testForceNumericInput(element, done) {
+        view.on('preventDefault', function () {
```

Comment:
You cannot use `view.on('preventDefault',` because it is not a View event / notification.